doc/source/whatsnew/v0.25.0.rst

```diff
@@ -100,6 +100,28 @@ Other Enhancements
 - :class:`DatetimeIndex` and :class:`TimedeltaIndex` now have a ``mean`` method (:issue:`24757`)
 - :meth:`DataFrame.describe` now formats integer percentiles without decimal point (:issue:`26660`)
 
+.. _whatsnew_0250.dependencies:
```

Comment:
can you merge this with the existing section
```
Increased minimum versions for dependencies
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
these as part of the numpy 1.13 bump

Reply:
Whoops forgot about that.